nzedb/Logger.php

```diff
@@ -533,14 +536,17 @@ private function initiateLog()
 	 * Rotate log file if it exceeds a certain size.
 	 *
 	 * @access private
+	 * @throws \nzedb\LoggerException
 	 */
 	private function rotateLog()
 	{
 		// Check if we need to rotate the log if it exceeds max size..
 		$logSize = filesize($this->logPath);
 		if ($logSize === false) {
 			return;
-		} else if ($logSize >= ($this->maxLogSize * 1024 * 1024)) {
+		}
+
+		if ($logSize >= ($this->maxLogSize * 1024 * 1024)) {
```

Comment:
Refactoring this should be to remove the additional return point to lower code complexity